src/aind_metadata_service/sharepoint/nsb2023/mapping.py

```diff
@@ -1747,7 +4404,11 @@ def aind_headpost_type(self) -> Optional[HeadPostType]:
                 self._nsb.headpost_type.WHC_NP: HeadPostType.WHC_NP,
                 self._nsb.headpost_type.WHC_NP_ZIRCONIA: HeadPostType.WHC_NP_ZIRCONIA,
                 self._nsb.headpost_type.AI_STRAIGHT_BAR_WELL: HeadPostType.AI_STRAIGHT_BAR_WELL,
-                self._nsb.headpost_type.OTHER_SEE_REQUESTOR_COMME: None,
+                self._nsb.headpost_type.LC_WELL: None,
```

Comment:
what's the difference between headpost and headpost type?

Reply:
and why are these being mapped to None while the ones above are not?

Reply:
As mentioned in earlier comment, there are 2 different fields in the request form. I believe "Headpost type" actually refers to the Well. These are being mapped to None right now because they are new options in this latest iteration. I was planning on going through the new fields and options with you in another ticket, but we can do it here if it's easy.